### ISSUE TITLE
Basic config for bspwm

### ISSUE THREAD

**Issue by SAVE-POlNT** [archlinux/archinstall#1311]
Hello , i would suggest to add a basic config of bspwm , 
at least start polybar sxhkd , dunst , install alacritty and set it in sxhkd and same for rofi

i don't know if this is set to other window managers or not 
a session manager and a wifi manager would be a nice plus to polybar

**Comment by dylanmtaylor** [comment]
> I'm not an frequent user or this profile myself, but it sounds reasonable to at least make a terminal work and if NetworkManager is chosen we do install nm-applet, in which case we should probably configure it to be visible if it's not by default :)

Part of me feels that bspwm configuration is going to be too individual and personal to make everyone happy. I'm not sure how that can be rectified. There's not really a default, agreed upon terminal emulator for most tiling window managers. I do think nm-applet is reasonable though in the case the user asked to use network manager, but people on desktops with  Ethernet probably won't need that so it shouldn't be in the base package list for the profile.

**Comment by SAVE-POlNT** [comment]
i think alacritty will make a good one , at least to start , and then the user would be able to modify whatever he wants , for the rest , it is in fact individual , the bar ,the runner ... 

either using the most common, polybar , rofi/dmenu , or leave just a terminal , which in this case will be same as using tty i guess lol

**Comment by SAVE-POlNT** [comment]
> > I'm not an frequent user or this profile myself, but it sounds reasonable to at least make a terminal work and if NetworkManager is chosen we do install nm-applet, in which case we should probably configure it to be visible if it's not by default :)
> 
> Part of me feels that bspwm configuration is going to be too individual and personal to make everyone happy. I'm not sure how that can be rectified. There's not really a default, agreed upon terminal emulator for most tiling window managers. I do think nm-applet is reasonable though in the case the user asked to use network manager, but people on desktops with  Ethernet probably won't need that so it shouldn't be in the base package list for the profile. 

i agree  for nm-applet , even if i forget sometimes to install it with additional packages , i connect it with my phone via usb-c or ethernet if available but maybe most ppl don't know about it

maybe we can detect if it's a desktop or not to install it with a power manager too

**Comment by Torxed** [comment]
> maybe we can detect if it's a desktop or not to install it with a power manager too

We do this already, as long as the profile returns `True` for `Profile.is_desktop` :) So that should be in the latest version.

**Comment by Torxed** [comment]
@svartkanin We still lack a basic terminal which was requested by OP.
We should probably fix that hehe.

**Comment by Torxed** [comment]
I thought so too hehe.

**Comment by Mehdi-YC** [comment]
still missing , i think , if someone can guide me a little bit so i can help with this , i noticed also that awesomewm do not have a display manager , same i know that everyone have to install his proper one , but maybe we can add it as an option or just use a servey to know what people use the most form a terminal and sisplay manager and use them for all the window managers
